http_service/bugbug_http/app.py

```diff
@@ -284,6 +285,28 @@ def get_result(job):
     return None
 
 
+def compress_response(data: dict, status_code: int):
+    """Compress data using gzip compressor and frame response
+
+    :param data: data
+    :type data: dict
+    :param status_code: response status code
+    :type status_code: int
+    :return: response with gzip compressed data
+    :rtype: Response
+    """
+
+    gzip_buffer = gzip.compress(orjson.dumps(data), compresslevel=9)
+
+    response = Response(status=status_code)
+    response.set_data(gzip_buffer)
+    response.headers["Content-Encoding"] = "gzip"
+    response.headers["Content-Length"] = len(response.get_data())
```

Comment:
Nit: You can probably just do:
```suggestion
    response.headers["Content-Length"] = len(gzip_buffer)
```

Reply:
Fixed

Reply:
Resolved